src/scale.ts

```diff
@@ -123,22 +118,14 @@ export function scaleTypePrecedence(scaleType: ScaleType): number {
   return SCALE_PRECEDENCE_INDEX[scaleType];
 }
 
-export const CONTINUOUS_TO_CONTINUOUS_SCALES: ScaleType[] = [
-  'linear',
-  'bin-linear',
-  'log',
-  'pow',
-  'sqrt',
-  'time',
-  'utc'
-];
+export const CONTINUOUS_TO_CONTINUOUS_SCALES: ScaleType[] = ['linear', 'log', 'pow', 'sqrt', 'symlog', 'time', 'utc'];
 const CONTINUOUS_TO_CONTINUOUS_INDEX = toSet(CONTINUOUS_TO_CONTINUOUS_SCALES);
 
+// FIXME: bin-ordinal is also continuous to discrete
```

Comment:
Not really, the input is already discrete.  In any case, we probably shouldn't leave this FIXME.

Reply:
bin-ordinal takes as input quantitative values, I thought.

Reply:
I think you're right.  Why don't you update it so we don't have this FIXME left.

Reply:
I removed the fixme since the domain you have to assign to the scale is still discrete.